.github/dependabot.yml

```diff
@@ -3,10 +3,10 @@ updates:
 - package-ecosystem: gomod
   directory: "/"
   schedule:
-    interval: daily
-  open-pull-requests-limit: 10
+    interval: weekly
+  open-pull-requests-limit: 5
   reviewers:
-  - aisuko
-  - Revolyssup
+    - aisuko
```

Comment:
For as much as we ❤️ aisuko, he's not actively performing this duty any longer and can be removed from this list.